src/api/docker-compose.yml

```diff
@@ -106,3 +106,52 @@ services:
       - 'traefik.http.routers.login.rule=Host(`login.localhost`)'
       # Specify the login port
       - 'traefik.http.services.login.loadbalancer.server.port=8080'
+      # Add middleware to this route to strip the /v1/posts prefix
```

Comment:
This looks wrong.  Can you rebase?  I've updated how the docker-compose files are structured.

Reply:
Yeah, this was the result of the merge conflicts resolution thing I was fixing. Will update